features/password_reset.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Conversation
+    As a user 
+    If I lose my password
+    I should be able to request a password reset
+
+    Background: User is Logged in and sends message
+        Given the following user is in database
+        | email              | password   | name |
+        | tore@example.com   | foobarbar  | Tore |
+        | olof@example.com   | mypassword | Olof |
+    
+    Scenario: As I user I shold be able to request a password reset
+      Given I am on the landing page
+      And I click on the "Login" button
+      And I click on the "Forgot your password?" button
+      And I fill in "Email" with "tore@example.com"
+      And I click on the "Send me reset password instructions" button
+      And I follow the password reset link
+      And I fill in "New password" with "newpassword"
+      And I fill in "Confirm new password" with "newpassword"
+      And I click on the "Change my password" button
+      Then I should see a message saying "Your password has been changed successfully. You are now signed in."
+
+
+
```

Comment:
Feature name Conversation, is that descriptive for this?
User story not really following convention, I referring to line 3. Please revisit.
Go through and correct grammar and spelling - line 7, 12,